<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SampleParams</class>
 <widget class="QWidget" name="SampleParams">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>202</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="theFrame">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame:focus {
	background-color: rgb(0, 170, 200);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="margin">
       <number>9</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QSlider" name="volumeSlider">
          <property name="toolTip">
           <string>Volume</string>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="muteButton">
          <property name="toolTip">
           <string>Mute</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="images.qrc">
            <normaloff>:/mute.png</normaloff>:/mute.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>12</width>
            <height>12</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QDial" name="panDial">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Pan</string>
          </property>
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="notchTarget">
           <double>10.000000000000000</double>
          </property>
          <property name="notchesVisible">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="noLoopRadio">
          <property name="text">
           <string>No loop</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="simpleLoopRadio">
          <property name="text">
           <string>Simple loop</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="autoLoopRadio">
          <property name="text">
           <string>Auto loop</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="namLabel">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>sample name</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="filenameLabel">
           <property name="font">
            <font>
             <italic>true</italic>
            </font>
           </property>
           <property name="text">
            <string>sample file</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="durationLabel">
           <property name="text">
            <string>duration</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QComboBox" name="keyCombo"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
